config/locales/en/steps.yml

```diff
@@ -382,6 +396,14 @@ en:
           remove_button: Remove owner %{index}
           deleted_flash: The owner has been deleted
 
+      properties_summary:
+        edit:
+          page_title: Assets summary list
```

Comment:
Content wants the title to match the heading here, is that possible?
https://docs.google.com/spreadsheets/d/1gdDpuyqWO6MJgUT1SOy0nbHd8AAdp1YebBzh1WDG040/edit#gid=0&range=D154

Reply:
I'm not sure this is possible. If not - let's compromise and have `Your client’s assets`